scripts/datasets/machine_translation/wmt2014_ende.sh

```diff
@@ -0,0 +1,73 @@
+SUBWORD_ALGO=$1
+SRC=en
+TGT=de
+SAVE_PATH=wmt2014_ende
+
+# Fetch the raw text
+nlp_data prepare_wmt \
+        --dataset wmt2014 \
+        --lang-pair ${SRC}-${TGT} \
+        --save-path ${SAVE_PATH}
```

Comment:
@szhengac This is our refactored script for preparing the WMT2014 dataset. `nlp_data` download the raw data. `nlp_preprocess clean_tok_para_corpus` cleans + tokenizes the corpus. `nlp_preprocess learn_subword` learns the subword, `nlp_preprocess apply_subword` applies the learned subword. No perl required.